common/util/debug.py

```diff
@@ -7,6 +7,7 @@
 
 _log = []
 enabled = False
+encoding_not_utf8 = "{} was sent as binaries and we dont know the encoding, not utf-8"
```

Comment:
Strings are always immutable, and names can always be overridden, no matter the casing. It's only a convention for readability sake.